acceptance/tests/external_ca_support/fixtures/auth.conf

```diff
@@ -1,60 +1,149 @@
-# Puppet 4.0.0 auth.conf, modified to allow requests from example.org for
+# Puppetserver 2.2.0 auth.conf, modified to allow requests from example.org for
 # external ca testing.
 
-# allow nodes to retrieve their own catalog
-path ~ ^/puppet/v3/catalog/([^/]+)$
-method find
-allow *.example.org
-allow $1
-
-# allow nodes to retrieve their own node definition
-path ~ ^/puppet/v3/node/([^/]+)$
-method find
-allow *.example.org
-allow $1
-
-# allow all nodes to access the certificates services
-path /puppet-ca/v1/certificate_revocation_list/ca
-method find
-allow *
-
-# allow all nodes to store their own reports
-path ~ ^/puppet/v3/report/([^/]+)$
-method save
-allow *.example.org
-allow $1
-
-# Allow all nodes to access all file services; this is necessary for
-# pluginsync, file serving from modules, and file serving from custom
-# mount points (see fileserver.conf). Note that the `/file` prefix matches
-# requests to both the file_metadata and file_content paths. See "Examples"
-# above if you need more granular access control for custom mount points.
-path /puppet/v3/file
-allow *
-
-### Unauthenticated ACLs, for clients without valid certificates; authenticated
-### clients can also access these paths, though they rarely need to.
-
-# allow access to the CA certificate; unauthenticated nodes need this
-# in order to validate the puppet master's certificate
-path /puppet-ca/v1/certificate/ca
-auth any
-method find
-allow *
-
-# allow nodes to retrieve the certificate they requested earlier
-path /puppet-ca/v1/certificate/
-auth any
-method find
-allow *
-
-# allow nodes to request a new certificate
-path /puppet-ca/v1/certificate_request
-auth any
-method find, save
-allow *
-
-# deny everything else; this ACL is not strictly necessary, but
-# illustrates the default policy.
-path /
-auth any
+authorization: {
```

Comment:
Going with tk-auth instead of the Ruby auth.conf makes sense to me.  For Puppet 5, Puppet Server will default to using tk-auth.  So long as these tests are only run with Puppet Server 5+ code, should be okay.  If these might run against older Puppet Server builds, though, (like 2.7.2, as I believe the agent pipeline currently does) you might want to have the test explicitly set `use-legacy-auth-conf` to `false`.  The older Puppet Server builds default the `use-legacy-auth-conf` setting to `true`.

You'll see lines like this in the `allow_arbitrary_node_name...` and `check_woy_cache_works` tests:

```ruby
modify_tk_config(master, options['puppetserver-config'], {'jruby-puppet' => {'use-legacy-auth-conf' => false}})
```